legal-api/migrations/versions/33501a263f32_sp_gp.py

```diff
@@ -0,0 +1,58 @@
+"""sp_gp
+
+Revision ID: 33501a263f32
+Revises: 94586877d4bc
+Create Date: 2022-02-28 12:53:01.294444
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '33501a263f32'
+down_revision = '94586877d4bc'
```

Comment:
Will update down revision if the pr (https://github.com/bcgov/lear/pull/1436) get merged